manifests/params.pp

```diff
@@ -38,8 +38,8 @@
   $framework_config = {
     'framework.server.name'     => $::fqdn,
     'framework.server.hostname' => $::fqdn,
-    'framework.server.port'     => '4440',
```

Comment:
I don't  think we can just remove these options.
I think we can be a bit more clever and do some merging on this hash depending on if ssh is enabled or not.